members/phillane/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<!--
+STYLING CREDIT TO: Harry Delugach's personal website
```

Comment:
In this case, you should also put this in your CSS sheet as well, since that is the main part where you are including his contributions.